recipes/cis_interface/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "cis_interface" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b815bbb0f9fa38d9e8f0f9e6a47bb4bfd67dbb9f5b6cdd81288dfe84a612e59a
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - czmq
+    - pip
+    - python
+    - sysv_ipc  # [unix]
+    - zeromq
```

Comment:
Please add the same requirements to run here as well, I already saw that you have some conflicts regarding you dependencies, to avoid that please add in this section the same libraries which you need to run it

Reply:
I'm not sure what you mean, these are the same libraries I need to run it. The conflicts were for czmq (of which I am the feedstock maintainer) and zeromq following the update with cb3 to build windows on both vc15 and vc9 (czmq currently segfaults on vc9, but zeromq was bumped to a new version to fix issues and the new version is required by pyzmq). I am awaiting a reply from the maintainers of the czmq package to see about a fix, but will skip building on vc9 for now.